app/src/main/java/io/homeassistant/companion/android/settings/widgets/ManageWidgetsViewModel.kt

```diff
@@ -1,72 +1,75 @@
-package io.homeassistant.companion.android.settings.widgets

```

Comment:
This file has become hard to review as it looks like the entire file was changed but that does not seem to be the case. Can you update this so the diff shows the actual changes done to the file please?

Reply:
Yes I agree with you, what I have done is runned KtLint on the project and it change the format but not the content I'll have a look into that 🤓

Reply:
(this is usually caused by changing the line endings on the file)

Reply:
That's correct, seems that the KtLint decided to add an ending line. I'll try to revert it

Reply:
Yes that's because the ending line. When I added the graphwidget I had to apply KTLint and that triggers the format of the whole time due not having the ending line.

Reply:
No, it is because you're changing between CRLF and LF. You should be able to change it back / manually add a line at the end without changing everything else.

Reply:
Nope doesn't seem to work

Reply:
I found that it was not the ending line or the CRLF.. I try that multiple times.. seems somehow I change the permissions of the file. I was tracking permissions with git, so I have revert it and stop tracking this changes. Please have another look